node/node.go

```diff
@@ -399,6 +409,21 @@ func (node *Node) getTransactionsForNewBlock(
 
 // StartServer starts a server and process the requests by a handler.
 func (node *Node) StartServer() {
+	for i := 0; i < RxWorkers; i++ {
+		go node.rxQueue.HandleMessages(node)
+	}
+
+	// start the goroutine to receive client message
+	// client messages are sent by clients, like txgen, wallet
+	go node.receiveGroupMessage(node.clientReceiver)
+
+	// start the goroutine to receive group message
+	go node.receiveGroupMessage(node.shardGroupReceiver)
```

Comment:
TODO @harmony-ek – plumb multiple queues/worker pools; avoid starvation due to client flooding

Reply:
#1725 fixes this; merging from master to pick it up.